.github/workflows/pr-close-terraform-prod-env.yaml

```diff
@@ -81,11 +82,11 @@ jobs:
       run: |
         OUTPUTS=$(python ./.github/scripts/tf-deploy-env.py --workspace prod --output)
         echo $OUTPUTS
-        TEST_ROLE_ARN=$(jq -r .role_arn.value <<< $OUTPUTS)
+        ASSUMED_ROLE_ARN=$(jq -r .role_arn.value <<< $OUTPUTS)
         DB_KEY_ARN=$(jq -r .jump_ec2_arn.value <<< $OUTPUTS)
         DB_CREDS_ARN=$(jq -r .db_creds_arn.value <<< $OUTPUTS)
         DB_HOST=$(jq -r .db_cluster_endpoint.value <<< $OUTPUTS)
         EC2_HOST=$(jq -r .jump_host.value <<< $OUTPUTS)
 
         chmod +x ./.github/scripts/aws_db_setup.sh
-        ./.github/scripts/aws_db_setup.sh $TEST_ROLE_ARN $DB_KEY_ARN $DB_CREDS_ARN $DB_HOST $EC2_HOST
+        ./.github/scripts/aws_db_setup.sh $ASSUMED_ROLE_ARN $DB_KEY_ARN $DB_CREDS_ARN $DB_HOST $EC2_HOST
```

Comment:
these args will need to be in the ENV to work since you removed the code in the script that handles them as command line args